lib/node_modules/@stdlib/blas/ext/base/dasumpw/src/addon.c

```diff
@@ -0,0 +1,117 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/ext/base/dasumpw.h"
+#include <node_api.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <assert.h>
+
+/**
+* Computes the sum of absolute values (L1 norm) of double-precision floating-point strided array elements using pairwise summation.
+*
+* ## Notes
+*
+* -   When called from JavaScript, the function expects three arguments:
+*
+*     -   `N`: number of indexed elements
+*     -   `X`: input array
+*     -   `stride`: stride length
+*/
+napi_value node_dasumpw( napi_env env, napi_callback_info info ) {
```

Comment:
This file should be simplified as part of this refactoring to make use of the available macros, please consult the linked PRs for how this should look like.